app/lib/ui/views/welcome/welcome.dart

```diff
@@ -0,0 +1,40 @@
+import 'package:app/core/view_models/welcome_view_model.dart';
+import 'package:app/ui/views/base_view.dart';
+import 'package:app/ui/views/signin/sign_in.dart';
+import 'package:app/ui/widgets/sidebar.dart';
+import 'package:flutter/material.dart';
+
+class Welcome extends StatelessWidget {
+  final String title;
+  const Welcome({Key key, this.title}) : super(key: key);
+  static const route = '/';
+  @override
+  Widget build(BuildContext context) {
+    return BaseView<WelcomeViewModel>(
+      builder: (context, model, child) => Scaffold(
+        appBar: AppBar(
+          title: Text('hello'),
+        ),
+        drawer: SideBar(),
+        body: Center(
+          child: Column(
+            mainAxisAlignment: MainAxisAlignment.center,
+            children: [
+              Text(
+                "Welcome",
+                key: Key('welcome'),
+              ),
+              // TODO: Register button
+              RaisedButton(
+                onPressed: () {
+                  Navigator.of(context).pushNamed(SignInScreen.route);
```

Comment:
- [ ] Should we not be using the service here?

Reply:
Ideally, yes, we should be using services across the application. But as I already mentioned above, refactoring everything in the application isn't the purpose of this commit. The purpose is to refactor the file structure and include some get_it implementation. Since this is not our final code (still in experimentation phase), I don't see the point of refactoring everything